examples/medplum-eligibility-demo/src/components/actions/UpdateCoverageEligibilityStatus.tsx

```diff
@@ -33,21 +28,21 @@ export function UpdateCoverageEligibilityStatus(props: UpdateCoverageEligibility
     handlers.close();
   };
 
-  const handleStatusUpdate = async (
-    coverageEligibility: CoverageEligibilityRequest | CoverageEligibilityResponse,
-    status: Coding
-  ): Promise<void> => {
-    const coverageEligibilityId = coverageEligibility.id as string;
+  const handleStatusUpdate = async (coverageEligibility: CoverageEligibilityRequest, status: Coding): Promise<void> => {
+    const statusCode = status.code as CoverageEligibilityRequest['status'];
 
-    // We use a patch operation here to avoid race conditions. This ensures that if multiple users try to add a note simultaneously, only one will be successful.
```

Comment:
@ksmith94 there was some miscommunication with the ticket. I want to keep the `patchResource` in place here, since it has concurrency benefits.

The change I wanted to make was in the `example-data.json`